src/components/Button.jsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react'
+import { connect } from 'react-redux';
+
+class Button extends React.Component {
+  constructor(props) {
+    super(props)
+
+    this.handleClear = this.handleClear.bind(this)
+  }
+
+  handleClear() {
+    const { whoMove, addIndexX, addIndexO, info } = this.props
+    const cells = Array(9).fill(null);
+    const move = 'X';
+    const infoAboutMove = 'Ход: X'
+    const counter = 0;
+
+    whoMove(cells, move, counter)
```

Comment:
It is not obvious what is going here and in other functions like this. Suggestion: move logical things to `redux`. Keep only UI in the components